app/_kong_plugins/openid-connect/index.md

```diff
@@ -554,6 +554,86 @@ rows:
+
+See the [cert-bound configuration example](/plugins/openid-connect/examples/cert-bound-access-tokens/) for more detail.
+
+### Demonstrating Proof-of-Possession (DPoP)
+
+Demonstrating Proof-of-Possession (DPoP) is an alternative technique to the [mutual TLS certificate-bound access tokens](#mutual-tls-client-authentication). Unlike its alternative, which binds the token to the mTLS client certificate, it binds the token to a JSON Web Key (JWK) provided by the client.
+
+<!--vale off-->
+{% mermaid %}
+sequenceDiagram
+    autonumber
+    participant client as Client <br>(e.g. mobile app)
+    participant kong as API Gateway <br>(Kong)
+    participant upstream as Upstream <br>(backend service,<br> e.g. httpbin)
+    participant idp as Authentication Server <br>(e.g. Keycloak)
+    activate client
+    client->>client: generate key pair
+    client->>idp: POST /oauth2/token<br>DPoP:$PROOF
+    deactivate client
+    activate idp
+    idp-->>client: DPoP bound access token ($AT)
+    activate client
+    deactivate idp
+    client->>kong: GET https://example.com/resource<br>Authorization: DPoP $AT<br>DPoP: $PROOF
+    activate kong
+    deactivate client
+    kong->>kong: validate $AT and $PROOF
+    kong->>upstream: proxied request <br> GET https://example.com/resource<br>Authorization: Bearer $AT
+    deactivate kong
+    activate upstream
+    upstream-->>kong: upstream response
+    deactivate upstream
+    activate kong
+    kong-->>client: response
+    deactivate kong
+{% endmermaid %}
+<!--vale on-->
+
+You can use the Demonstrating Proof-of-Possession option without mTLS, and even with plain HTTP, although HTTPS is recommended for enhanced security.
+
+When verification of the DPoP proof is enabled, {{site.base_gateway}} removes the `DPoP` header and changes the token type from `dpop` to `bearer`.
+This effectively downgrades the request to use a conventional bearer token, and allows an OAuth2 upstream without DPoP support to work with the DPoP token without losing the protection of the key binding mechanism.
+
+DPoP is compatible with the following authentication methods:
+
+* [JWT Access Token authentication](#jwt-access-token-authentication-flow)
+* [Introspection authentication](#introspection-authentication-flow)
+* [Session authentication](#session-authentication-workflow)
+
+Session authentication is only compatible with DPoP when used along with one of the other supported authentication methods:
+* If multiple `openid-connect` plugins are configured with the `session` authentication method, we strongly recommend configuring different values of [`config.session_secret`](/plugins/openid-connect/reference/#schema--config-session-secret) across plugin instances for additional security. This avoids sessions being shared across plugins and possibly bypassing the proof of possession validation.
```

Comment:
```suggestion
Session authentication is only compatible with DPoP when used along with one of the other supported authentication methods. If multiple `openid-connect` plugins are configured with the `session` authentication method, we strongly recommend configuring different values of [`config.session_secret`](/plugins/openid-connect/reference/#schema--config-session-secret) across plugin instances for additional security. This avoids sessions being shared across plugins and possibly bypassing the proof of possession validation.
```
There was only one item in the list, so I think this edit is correct, but it might be you missed a bullet or two when copy/pasting.

Reply:
There was only one bullet.